4_Bayes_Rule/Bayes_Rule_All.ipynb

```diff
@@ -0,0 +1,4120 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    # Simulate 286 visits
</pre><ul><li>n_visitors says 1000, not 286</li><li>I love the distinction between visits and conversions -- great insights, it'll make way easier to understand for students!</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>